doc/tutorial/climada_hazard_Bushfire-Copy1.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
I suppose this document is not needed?

Reply:
removed it. will do some further work on the tutorial anyway